ui/src/app/applications/components/filter/filter.tsx

```diff
@@ -31,6 +32,17 @@ export const CheckboxRow = (props: {value: boolean; onChange?: (value: boolean)
         setValue(props.value);
     }, [props.value]);
 
+    const tooltipProps = {
+        placement: 'top' as any,
+        popperOptions: {
+            modifiers: {
+                preventOverflow: {
+                    boundariesElement: 'window' as any
+                }
+            }
+        }
+    };
```

Comment:
I would push to avoid `any` type declarations when possible, here we can delare this type as `Partial<React.ComponentProps<typeof tooltip>>` to remove the type error

```suggestion
    const tooltipProps: Partial<React.ComponentProps<typeof Tooltip>> = {
        placement: 'top',
        popperOptions: {
            modifiers: {
                preventOverflow: {
                    boundariesElement: 'window'
                }
            }
        }
    };
```